### DISCUSSION TITLE
Foods Tag

### DISCUSSION THREAD

**Discussion by TheBedrockMaster** [vectorwing/FarmersDelight#1056] [Ideas]
Please add the `c:foods` item tag to edible items from the mod that are missing it.

**Comment by vectorwing** [comment]
FD already adds most, if not all, of its food items to NeoForge's `foods` tag conventions.
Is there something specific that we're missing? Feel free to suggest new tagging here.

**Comment by Roelymole** [comment]
The various pies should be added to the `#c:foods/pie` item tag.

Also, the Milk Bottle item (and possibly Hot Cocoa) should be added to the `#c:drinks/milk` item tag and the Melon Juice item (and possibly Apple Cider) should be added to the `#c:drinks/juice` item tag.

The Hot Cocoa and Apple Cider items should definitely be part of the `#c:drinks` item tag or one of its subtags, but it is less clear-cut.
